source/simulator/core.cc

```diff
@@ -130,7 +130,9 @@ namespace aspect
             this_mpi_process(mpi_communicator)
             == 0)),
 
-    computing_timer (pcout, TimerOutput::never,
+    computing_timer (mpi_communicator,
+                     pcout,
```

Comment:
No, if you don't specify a communicator, it will do timing on processor 0 only. With this change we do barriers in each timing section.